assets/css/main.scss

```diff
@@ -9,3 +9,4 @@
 @import "about";
 @import "gallery";
 @import "syntax";
+@import "components/footer";
```

Comment:
Maybe we can document this briefly in the README so people know they can re-use these variables?

Reply:
I've added a note to the README and pushed.